File: DSA/DivyanshKumarSingh_590012175/Day_16_Question_3.c
#include <stdio.h>

int findMax(int arr[], int size) 
{
    if (size == 0) 
    {
        return -1; 
    }

    int max = arr[0]; 

    for (int i = 1; i < size; i++) 
    {
        if (arr[i] > max) 
        {
            max = arr[i];
        }
    }

    return max;
}

int main() 
{
    int n;

    printf("Enter number of elements: ");
    scanf("%d", &n);

    if (n <= 0) {
        printf("Array is empty.\n");
        return 0;
    }

    int arr[n];

    printf("Enter %d integers: ", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int max = findMax(arr, n);
    printf("Maximum element: %d\n", max);

    return 0;
}

File: DSA/DivyanshKumarSingh_590012175/Day_16_Question_3.py
def findMax(arr):
    if not arr:
        return None  

    max_value = arr[0]

    for num in arr[1:]:
        if num > max_value:
            max_value = num

    return max_value

inp = input("Enter integers separated by spaces: ")
arr = list(map(int, inp.strip().split()))

maximum = findMax(arr)
if maximum is not None:
    print("Maximum element:", maximum)
else:
    print("Array is empty.")
